zeppelin/zeppelin.sh

```diff
@@ -45,6 +45,16 @@ function install_zeppelin(){
   if [ $? != 0 ]; then
     err 'Failed to install zeppelin'
   fi
+
+  # If both asm-3.1.jar and asm-5.0.4.jar are found in /usr/lib/zeppelin/lib for
+  # Zeppelin 0.7, delete asm-5.0.4.jar. This is a temporary workaround before
+  # we figure out the root cause of asm conflict.
+  version=$(dpkg --status zeppelin | grep 'Version:')
```

Comment:
`configure_zeppelin` has a slightly cleaner way to get and compare the version information